guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>In this guide, we have learned about the analysis capabilities available in the <code>arcgis.geoanalytics</code> module and how some of the common concepts, such as environment settings, sync operation, filer etc., can be applied across all tools. In the next guide, we will learn in more detail about the tools available in the <code>arcgis.geoanalytics.summarize_data</code> submodule.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>